aries_cloudagent/resolver/default/indy.py

```diff
@@ -50,6 +50,12 @@ async def _resolve(self, profile: Profile, did: str) -> dict:
         except LedgerError as err:
             raise DIDNotFound(f"DID {did} could not be resolved") from err
 
+        did_doc = self._did_doc_builder(did, endpoint, recipient_key)
+
+        return did_doc.serialize()
+
+    def _did_doc_builder(self, did, endpoint, recipient_key):
```

Comment:
Similar to my previous comment, unless we intend to use this method elsewhere, I think it would be clearer to just inline the body instead of creating a separate method.